tests/frontend/test_blog_homepage.py

```diff
@@ -0,0 +1,63 @@
+
+import pytest
+from selenium.webdriver.common.by import By
+
+from pages.desktop.frontend.blog_homepage import BlogHomepage
+from pages.desktop.frontend.home import Home
+
+
+@pytest.mark.nondesstructive
+def test_header_logo_button(base_url, selenium):
+    selenium.get(f'{base_url}/blog/')
+    page = BlogHomepage(selenium, base_url).wait_for_page_to_load()
+    page.header.click_title()
+    homepage = Home(selenium, base_url)
+    assert homepage.hero_banner
+
+
+@pytest.mark.nondesstructive
+def test_articles_elements_are_displayed(base_url, selenium):
+    selenium.get(f'{base_url}/blog/')
+    page = BlogHomepage(selenium, base_url).wait_for_page_to_load()
+    for article in page.articles:
+        assert article.image.is_displayed()
+        assert article.title
+        assert article.date
+        assert article.intro_text
+        assert article.read_more_link
+
+
+@pytest.mark.nondesstructive
+def test_click_article_image(base_url, selenium):
```

Comment:
a more suggestive name for the purpose of this test might be: `test_open_article_by_clicking_article_image`;

Reply:
Done.